### PR TITLE
CB-1513 – remove environment-kerberosconfig relation from cloudbreak

### PR DESCRIPTION
**Pull request by gregito**
CB-1513 – remove environment-kerberosconfig relation from cloudbreak

**UI change/update is necessary!**

### CODE REVIEW

File: core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-- // CB-1513 remove kdc env relation
+-- Migration SQL that makes the change goes here.
+DROP TABLE IF EXISTS env_kdc;
```
Comment (attilapalfi92) [inline core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql]: This also drops the indexes that refer to this table right?

Comment (gregito) [reply core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql]: most likely

Comment (gregito) [reply core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql]: are you suggest that to drop the indexes beforehand?

File: core/src/main/java/com/sequenceiq/cloudbreak/service/kerberos/KerberosConfigService.java [hunk 2]
```diff
@@ -35,10 +35,14 @@
     private CloudbreakRestRequestThreadLocalService restRequestThreadLocalService;
 
     @Override
-    protected EnvironmentResourceRepository<KerberosConfig, Long> repository() {
+    protected WorkspaceResourceRepository<KerberosConfig, Long> repository() {
         return repository;
     }
 
+    @Override
+    protected void prepareDeletion(KerberosConfig resource) {
```
Comment (biharitomi) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/kerberos/KerberosConfigService.java]: This method has a proper implementation in the previously extended abstract class `AbstractEnvironmentAwareService`

Comment (gregito) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/kerberos/KerberosConfigService.java]: yes, but this class does not extend the `AbstractEnvironmentAwareService` anymore. But I'll make a proper one here also.

File: core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql (hunk 1 above)
Comment (attilapalfi92) [reply core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql]: only if it does not do it

Comment (gregito) [reply core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql]: it does

**Pull request merged**